storage-proofs/src/batchpost.rs

```diff
@@ -9,7 +9,7 @@ use serde::ser::Serialize;
 use crate::crypto::blake2s::blake2s;
 use crate::error::Result;
 use crate::hasher::{Domain, Hasher};
-use crate::merkle::MerkleTree;
+use crate::merkle::{MerkleTree, MmapStore};
```

Comment:
you could just make the `MerkleTree` be an alias to `MerkleTree<D, F, MmapStore<D>>` that way you don't have to change all the instances in the code base

Reply:
don't bother reviewing this at the moment, fyi, the issue is not concentrating all renames in a single place but whether we want to have *all* `MerkleTree`s using `mmap` instead of `Vec` (if the answer is yes, then we can do as you suggest, but then I would ask why did we implement a new `Store` trait in the first place).

Reply:
I see, that makes sense 👍